styles/git-blame.less

```diff
@@ -7,7 +7,6 @@
 
 @git-blame-bg-color: @base-background-color;
 
-atom-text-editor::shadow .git-blame-mount,
```

Comment:
based on the automatic upgrade rule, it should probably just be changed to `atom-text-editor.editor .git-blame-mount,` instead of deleted